src/librustdoc/clean/types.rs

```diff
@@ -1867,6 +1867,11 @@ impl Span {
         Self(sp.source_callsite())
     }
 
+    /// Unless you know what you're doing, use [`Self::from_rustc_span`] instead!
```

Comment:
I think `from_rustc_span` needs some docs so if people read that they can know why is it documented this way.

Reply:
That's a good point too. For reference, `from_rustc_span` changes the span from the macro definition to the macro callsite if it's inside a macro.